modules/special-rules/pages/air-transport-mission.adoc

```diff
@@ -27,20 +34,27 @@ NOTE: If a {transport-x} Flyer unit is destroyed while airborne then any embarke
 
 . Make ranged attacks with the Flyer units, just like you would with ground units in the Shooting phase -- use the Firepower values of the Flyer units and any super-heavy weapons that they have.
 But note:
-* You can draw a line of fire only where an enemy unit lies within range *and* inside the forward 90° arc of your Flyer unit.
-Ignore any intervening terrain though, as it is far below the altitude of the flyers.
-* Otherwise, the attack works just like a normal shooting attack -- so for example,as long as you meet the other requirements then you may split fire between multiple target detachments.
+* You can attack only where an enemy unit lies within range *and* inside the forward 90° arc of your Flyer unit.
+Flyer units do not need line of fire, as they can see over all intervening units and terrain.
+* Otherwise, the attack works just like a normal shooting attack -- so for example, as long as you meet the other requirements then you may split fire between multiple target detachments.
```

Comment:
> My only reservation relates to something I missed in the first draft and to the point I've just made — what if the flyer is a War Engine unit with weapon arc(s) other than Front 90°? Do we mean to say that it can only make attacks to the extent that its weapons operate within arcs that overlap with the forward 90° arc?

It's unlikely that a War Engine Flyer would have wider weapon arcs than a normal Flyer, given that War Engines normally have more restrictive arcs, but even if it did we'd deal with that in the War Engine section of the rules, like we do for other War Engine exceptions.

Reply:
The problem is that normal flyers don't even *have* weapon arcs (so by default I'd assume that they can shoot all round, like non- War Engine ground units) — there is just this rule that says they can only attack targets inside their forward 90° arc, in this context\*. Similar ideas, but not quite the same thing.

It doesn't really help us to define what should happen with War Engine units which, in principle, could have weapon systems on mounts with arcs like 'All Around (360°)' or 'Rear 180°' or 'Left 180°'...

So I'm really asking, which do we actually intend out of these two quite different interpretations: ?

1. **Non-** War Engine flyers are restricted to the forward 90° arc, at least for the attacks they make during Ground Attack/Air Assault — while War Engine flyers can use their usual arcs for their attacks.
2. **All** flyers are limited to the forward 90°, at least for the attacks they make during Ground Attack/Air Assault — and so in this context War Engine flyers can attack only to the extent that their arcs overlap with this arc.

*\*The rules only mention this arc for flyers in the context of completing attacks against ground units. The Intercept mission doesn't mention arcs at all. Do we assume 360° in that case, or 90°? Or, is there no edge case in which this would actually matter? Might it matter for some new missions and/or War Engine flyers that we invent...?*

Reply:
> The problem is that normal flyers don't even _have_ weapon arcs (so by default I'd assume that they can shoot all round, like non- War Engine ground units) — there is just this rule that says they can only attack targets inside their forward 90° arc, in this context*. Similar ideas, but not quite the same thing.

Do you mean regular ground units? Flyers have 90˚ arcs, as described in their rules.

> So I'm really asking, which do we actually intend out of these two quite different interpretations: ?
> 
> 1. **Non-** War Engine flyers are restricted to the forward 90° arc, at least for the attacks they make during Ground Attack/Air Assault — while War Engine flyers can use their usual arcs for their attacks.
> 2. **All** flyers are limited to the forward 90°, at least for the attacks they make during Ground Attack/Air Assault — and so in this context War Engine flyers can attack only to the extent that their arcs overlap with this arc.

Neither.

Flyers default to 90˚ weapon arcs, as described in the Flyer rules. _If_ War Engine Flyers have different weapon arcs, this will be covered in the rules for War Engine Flyers.

> *The rules only mention this arc for flyers in the context of completing attacks against ground units. The Intercept mission doesn't mention arcs at all. Do we assume 360° in that case, or 90°? Or, is there no edge case in which this would actually matter? Might it matter for some new missions and/or War Engine flyers that we invent...?

It's mentioned in the rules for attacks against ground units because that's the only context in which a weapon arc could apply. E40k Intercepts are completely abstracted dogfights with no concept of weapon arcs, range, model position etc. The only reason you even put the intercepting models on the table is to show that a dogfight is happening.

Were you thinking of EA?